src/spec_procs.cpp

```diff
@@ -5680,6 +5680,12 @@ SPECIAL(chargen_skill_annex) {
       return TRUE;
     }
 
+    if (IS_OTAKU(ch) && GET_SKILL(ch, SKILL_COMPUTER) < 6) {
+      send_to_char(ch, "Otakus must have at least a rating 6 in Computers, but your proficiency is only %d. You should ^WPRACTICE^n your Computers before proceeding.\r\n",
```

Comment:
If this is a hard requirement, can we make them start with 6 computers and -6 skill points by default, and refuse to let them create with any less than 6 skill points?